### DISCUSSION TITLE
Full or partial backups

### DISCUSSION THREAD

**Discussion by pezubi** [lavinir/hassio-onedrive-backup#167] [Ideas]
I have a quick question, I am unsure, whether the Add-on creates a partial or full backup.
I have some full and some partial backups and I can't find a reasoning to that, so I thought I'd just ask  :D
Therefor I suggest to change the ReadMe as to show, what kind of backup is done or to implement a switch or something to choose one of the options named above.

**Comment by lavinir** [comment]
Full / Partial backup definition is not something that is defined by the addon but conforms to the HA definition. 
If you excluded any addons / folder paths from the backup you will end up with a partial backup. If nothing is excluded you will get a full backup. 

The logs will tell you which is created. As long as you don't change the settings the addon will keep making the same 'type' of backup. (mind you, if a backup is created outside the addon it could be either depending on what was backed up)
